pkg/cmd/resources.go

```diff
@@ -36,3 +64,30 @@ Use "stripe [command] --help" for more information about a command.
 		ansi.Bold("Available commands:"),
 	)
 }
+
+func getAllPluginCommands() ([]string, error) {
+	var manifest PluginList
+
+	resp, err := http.Get(pluginManifestURL)
+	if err != nil {
+		return []string{}, err
+	}
+	defer resp.Body.Close()
+
+	respBytes, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return []string{}, err
+	}
+
+	err = toml.Unmarshal(respBytes, &manifest)
+	if err != nil {
+		return []string{}, err
+	}
+
+	var pluginCommands []string
+	for _, plugin := range manifest.Plugins {
+		pluginCommands = append(pluginCommands, plugin.Shortname)
+	}
+
+	return pluginCommands, nil
+}
```

Comment:
this feels like it belongs in the plugins package as a public function in `utilities.go`, and then `resources_test.go` can import and use this as a public function. This means we can also not worry about defining the plugin structs in this file nor the test file.